torch/testing/__init__.py

```diff
@@ -86,10 +86,17 @@ def make_non_contiguous(tensor):
     return input
 
 
-def get_all_dtypes():
-    return [torch.uint8, torch.int8, torch.int16, torch.int32, torch.int64,
-            torch.float16, torch.float32, torch.float64]
+def get_all_dtypes(withBool=False):
```

Comment:
This isn't idiomatic Python variable naming; use snake case (`with_bool`) rather than camel.